docs/source/user/aerodyn/input.rst

```diff
@@ -451,13 +454,18 @@ Since the hub and blades are joined elements, hub buoyancy should be turned on i
 
 Nacelle Properties
 ~~~~~~~~~~~~~~~~~~
-The input parameters in this section pertain to the calculation of buoyant loads
-on the nacelle and are only used when ``Buoyancy = TRUE``.
+The input parameters in this section pertain to the calculation of buoyant and drag loads
+on the nacelle and are only used when ``Buoyancy = TRUE`` or ``NacelleDrag = TRUE``.
 
 ``VolNac`` is the volume of the nacelle and ``NacCenB``` is the 
 position (x,y,z vector) of the nacelle center of buoyancy from
 the yaw bearing in local nacelle coordinates. To neglect buoyant 
-loads on the nacelle, set ``VolNac`` to 0.
+loads on the nacelle, set ``VolNac`` to 0. Only used when ``Buoyancy = TRUE``.
+
+``NacArea`` are the projected areas (x,y,z) of the nacelle in the nacelle coordinate system, 
+``NacCd`` is the drag coefficient for the nacelle areas defined by ``NacArea``and ``NacDragAC`` is the 
```

Comment:
Is a different Cd defined for each direction? If so, it would be good to clarify this here.

Reply:
Updated, hope it reads better now.